### DISCUSSION TITLE
Skydance WZ5 dimmer/rgbcct mode

### DISCUSSION THREAD

**Discussion by sttrophey** [Koenkk/zigbee2mqtt#21431] [Devices]
I have 4 devices and one of them detected only in dimmer mode. I tried to change modes (all of them) on device with no effect. Tried del and reconnect it. What i do wrong? Where i can change setting in manual mode?
<img width="1174" alt="Снимок экрана 2024-02-16 в 23 11 20" src="https://github.com/Koenkk/zigbee2mqtt/assets/76071480/ee608d15-9693-4cd9-8255-90f8d8b75eb3">

**Comment by pLavrenov** [comment]
remove from z2m, hold down the button on the case for 14 seconds when the green light comes on, after that you need to turn off the power and turn it on.
